### PR TITLE
fix: Adapt ensemble configs to work with loss refactor. 

### PR DESCRIPTION
**Pull request by jakob-schloer**
## Description

The default configs for the ensemble training were not working with the new loss refactor. This PR creates a `config/training/ensemble.yaml` that sets default values for training with ensemble members.

A new `ensemble.yaml` file was required because the `default.yaml` would set `mse` as a `training.validation_metrics`, which can be extended by other metrics but cannot be fully overwritten (mse would always be part of the validation_metrics). The MSE metric, however, is not compatible with the `GraphEnsForecaster` model. 

## Type of Change

-   [x] Bug fix (non-breaking change which fixes an issue)

## Code Compatibility

-   [x] I have performed a self-review of my code

### Code Performance and Testing

-   [x] I have tested the changes on a single GPU
-   [x] I have tested the changes on multiple GPUs / multi-node setups
-   [x] I have tested inference on a trained model

### Dependencies

-   [x] I have not introduced new dependencies in the inference portion of the pipeline

### CODE REVIEW

**Comment by anaprietonem** [comment]
thanks for this fix Jakob!

File: training/src/anemoi/training/config/debug_ens.yaml [hunk 1]
```diff
@@ -6,10 +6,10 @@ defaults:
 - hardware: example
 - graph: encoder_decoder_only
 - model: transformer_ens
-- training: default
+- training: ensemble
 - _self_
 
-config_validation: True
+config_validation: False
```
Comment (anaprietonem) [inline training/src/anemoi/training/config/debug_ens.yaml]: I'd recommend, keeping the config_validation to default `True` value

Comment (jakob-schloer) [reply training/src/anemoi/training/config/debug_ens.yaml]: Of course, sorry :)

File: training/src/anemoi/training/config/training/ensemble.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  ignore_nans: False
+  alpha: 1.0
+
+# Validation metrics calculation,
+# This may be a list, in which case all metrics will be calculated
+# and logged according to their name.
+# These metrics are calculated in the output model space, and thus
+# have undergone postprocessing.
+validation_metrics:
+  # loss class to initialise, can be anything subclassing torch.nn.Module
```
Comment (anaprietonem) [inline training/src/anemoi/training/config/training/ensemble.yaml:86]: Is there any reason why we go from 2 set of validation metrics to just one? (ie discarding the config section with alpha 0.95)

Comment (jakob-schloer) [reply training/src/anemoi/training/config/training/ensemble.yaml:86]: It's the difference between the almost fair kernel crps and kernel crps. I thought it was cleaner to show only one.

Comment (anaprietonem) [reply training/src/anemoi/training/config/training/ensemble.yaml:86]: thanks for the clarification!

**Review by anaprietonem — APPROVED** [review]
LGTM!
